### ISSUE TITLE
Add the ability to put image galleries in the documentation

### ISSUE THREAD

**Issue by pcon** [ManiacalLabs/Engravinator.com#17]
Add the ability to put images into the documentation annotated in such a way that they will be converted into a gallery

**Comment by pcon** [comment]
Assuming you put the images in the `docs/mk1/img/` folder the following markdown will now generate a gallery.

```markdown
---
photogallery: true
---

# This is a gallery of images

The images below will cycle through only the first two and will loop
<a href="/mk1/img/largephoto1.png" data-imagelightbox="groupa"><img src="/mk1/img/smallphoto1.png"></a>
<a href="/mk1/img/largephoto2.png" data-imagelightbox="groupa"><img src="/mk1/img/smallphoto2.png"></a>

The images below this are grouped separately and loop through it's own cycle.
<a href="/mk1/img/largephoto3.png" data-imagelightbox="groupb"><img src="/mk1/img/smallphoto3.png"></a>
<a href="/mk1/img/largephoto4.png" data-imagelightbox="groupb"><img src="/mk1/img/smallphoto4.png"></a>
```

Currently this only allows for swipe / keyboard navigation of photos.  Open up a new issue if it's desired to add navigation buttons to click on or to add additional styling to gallery images / lightboxed image.

**Comment by pcon** [comment]
@adammhaile Here you go.  Lemme know if you need more but this should unblock you
